lamassu-remote-install/install

```diff
@@ -97,11 +99,18 @@ echo $SEED > $SEED_FILE
 
 decho "Installing latest npm package manager for node..."
 retry 3 npm -g --unsafe-perm install npm@5  >> $LOG_FILE 2>&1
-NODE_MODULES=$(npm -g root)
-NPM_BIN=$(npm -g bin)
 
 decho "Installing lamassu-server..."
-retry 3 npm -g --unsafe-perm install lamassu/lamassu-server#${1-master} >> $LOG_FILE 2>&1
+mkdir -vp ${INSTALL_DIR} >> $LOG_FILE 2>&1
+cd ${INSTALL_DIR} >> $LOG_FILE 2>&1
+retry 3 npm --unsafe-perm install --production lamassu/lamassu-server#${1-master} >> $LOG_FILE 2>&1
```

Comment:
@joshmh please checkout this.
do you think it's fine if we install just production deps?
we don't really need ava or mocha on do right?